goiardi/plan.sh

```diff
@@ -0,0 +1,48 @@
+pkg_name=goiardi
+pkg_origin=core
+pkg_version=0.11.10
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_description="A Chef server written in Go"
+pkg_upstream_url="http://goiardi.gl"
+# pkg_scaffolding="some/scaffolding"
+pkg_source="https://github.com/ctdk/goiardi/archive/v${pkg_version}.tar.gz"
+# pkg_filename="${pkg_name}-${pkg_version}.tar.gz"
+pkg_shasum=74bb265eaef7d9d470a8ccce7bef651aaff41e314cd177a11ad6493ff0179345
+pkg_deps=(
+  core/glibc
+)
+pkg_build_deps=(
+  core/go
+  core/git
+)
+pkg_bin_dirs=(bin)
+# pkg_svc_run="haproxy -f $pkg_svc_config_path/haproxy.conf"
+# pkg_exports=(
+#   [host]=srv.address
+#   [port]=srv.port
+#   [ssl-port]=srv.ssl.port
+# )
+# pkg_exposes=(port ssl-port)
+# pkg_binds=(
+#   [database]="port host"
+# )
+# pkg_binds_optional=(
+#   [storage]="port host"
+# )
+# pkg_interpreters=(bin/bash)
+# pkg_svc_user="hab"
+# pkg_svc_group="$pkg_svc_user"
```

Comment:
I think we should probably remove the commented code unless it can be used directly as an example for someone looking to make a wrapper plan.